packages/components/src/components/field/field.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+type TFieldProps = {
+    className?: string;
+    message: string | ReactNode | boolean;
+    type: string;
+};
```

Comment:
```suggestion
import React from 'react';

type TFieldProps = {
    className?: string;
    message: string;
    type: 'error' | 'warn';
};
```
i agree with @farzin-fs